lumicks/pylake/detail/h5_helper.py

```diff
@@ -13,9 +13,10 @@ def write_h5(h5_file, output_filename, compression_level=5, omit_data=None):
         Output file name.
     compression_level : int
         Compression level for gzip compression.
-    omit_data : Optional[Set[str]]
+    omit_data : str or iterable of str, optional
         Which data sets to omit. Should be a set of h5 paths.
     """
+    omit_data = {omit_data} if isinstance(omit_data, str) else omit_data
```

Comment:
Nah, chose `Set` since that is what we initially documented as the input type, but could've taken any iterable really.

I don't have a particular preference for immutability or uniqueness. Can change it to a `tuple` if you prefer. Could even make it a dummy function that just yields the string to really drive the point home that it just needs to be an `iterable` that yields strings. Maybe that should explicitly be tested in a test actually.

Reply:
nah this is fine, I was just curious. If anything maybe use a tuple or list in the example in the docstring since sets might not be as well known to users